main.go

```diff
@@ -75,7 +93,7 @@ func (s *service) start() error {
 	listener, err := net.Listen("tcp", s.conf.Listen)
 
 	if err != nil {
-		return err
+		log.Fatalln(err)
```

Comment:
It's has specific meaning e.g. when port is already in use we can see message about it. When we only `return err` we hidden important error messages from net.